test/api/application/create.test.js

```diff
@@ -4,8 +4,44 @@ const { promisify: p } = require('util')
 const { Server, GroupId } = require('../../helpers')
 
 // const sleep = async (t) => new Promise(resolve => setTimeout(resolve, t))
+test('tribes.application.create (v3 application)', async t => {
+  const server = Server()
+  const adminIds = [keys.generate().id]
+  const groupId = GroupId()
+  const answers = [
+    { q: 'what is your favourate pizza flavour', a: 'hawaiian' }
+  ]
+
+  const profileId = '%FiR41bB1CrsanZA3VgAzoMmHEOl8ZNXWn+GS5vW3E/8=.sha256'
```

Comment:
how cautious do we want to be?
we could add a step on `application.create` which looks to see if the `profile` linked to : 
- has recps which includes `adminIds`
- has a `preferredName` `legalName` (or just `name`) field

Reply:
nah forget this